.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Test
+
+on:
+  push: { "branches": [ "master" ] }
+  pull_request: { "branches": [ "master" ] }
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install OS Packages
+      uses: mstksg/get-package@v1
+      with:
+        apt-get: libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev
```

Comment:
Minor remark but you'll likely not need any of these (except maybe `libgmp-dev`) for the coin-selection :thinking: I need most of them on the cardano-transactions library because of the dependencies with the ledger and consensus libs.

Reply:
Fixed in bd8668313c59197e83dcb0f8704468d2b44e5b68.